src/table/runtime.rs

```diff
@@ -11,12 +13,42 @@ use core::ptr;
 #[repr(C)]
 pub struct RuntimeServices {
     header: Header,
+    get_time:
+        unsafe extern "win64" fn(time: &mut Time, capabilities: *mut TimeCapabilities) -> Status,
+    set_time: unsafe extern "win64" fn(time: &Time) -> Status,
     // Skip some useless functions.
-    _pad: [usize; 10],
-    reset: extern "win64" fn(rt: ResetType, status: Status, data_size: usize, data: *const u8) -> !,
+    _pad: [usize; 8],
+    reset: unsafe extern "win64" fn(
+        rt: ResetType,
+        status: Status,
+        data_size: usize,
+        data: *const u8,
+    ) -> !,
 }
```

Comment:
Time management was used by EFI_FILE_PROTOCOL's Get/SetInfo, so I implemented it.